tests/test_bundle_download.py

```diff
@@ -159,20 +166,58 @@ def test_load_weights(self, bundle_files, bundle_name, repo, device, model_file)
 
                 # load instantiated model directly and test, since the bundle has been downloaded,
                 # there is no need to input `repo`
+                _model_2 = nets.__dict__[model_name](**net_args)
```

Comment:
```suggestion
                _model_2 = getattr(nets, model_name)(**net_args)
```

Very nitpicky but I think this would work.

Reply:
Thanks for the suggestion, updated it in https://github.com/Project-MONAI/MONAI/pull/6835/commits/63d3a845b92a643773e08a05482b939bbb88e20d.